posts/2021-11-30-RedHat-Connect-London.adoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "A day out in London at the RedHat Connect Summit"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/yasmin-aumeeruddy
+author_github: https://github.com/yasmin-aumeeruddy
+seo-title: A day out in London at the RedHat Connect Summit
+seo-description: 
+blog_description: 
+open-graph-image: https://openliberty.io/img/AndyMcCright-1.png
```

Comment:
need to change the open graph image to be one of the images from the post, in this format:

https://openliberty.io/img/blog/image-filename.png